### ISSUE TITLE
Can't join server on a vanilla client with mod installed

### ISSUE THREAD

**Issue by pepperonitony513** [The-Fireplace-Minecraft-Mods/Fireplace-Lib#54]
### Mod Version

7.0.2

### What mod loader(s) are you seeing the problem on?

Fabric

### Mod Loader Version

0.14.8

### Problem Description

Whenever I try to launch my 1.19 fabric server with fireplace lib I get this error.

Internal Exception: io.netty.handler.codec.DecoderException: aa: Non [a-z0-9/._-] character in path of location: minecraft:mi

Same thing happens when I use another fireplacelib mod (in this case unforgiving void), but I'm confused as to why I'm getting this error when it's just the library mod.

### Steps To Reproduce

1. Make fabric server
2. Include fireplacelib in mods 
3. Join on vanilla client

### Crash Report or Error Logs

```shell
Internal Exception: io.netty.handler.codec.DecoderException: aa: Non [a-z0-9/._-] character in path of location: minecraft:mi
```

**Comment by The-Fireplace** [comment]
Most likely related to #51

**Comment by purejosh** [comment]
Also getting this. I have a velocity proxy, with a modded fabric server in the proxy. Logging into it from the client (mods match on client and server) causes this error, probably since it's going through the proxy first.

**Comment by The-Fireplace** [comment]
Should be fixed in 7.0.3
